src/utils/types/ITransactions.ts

```diff
@@ -0,0 +1,23 @@
+export interface ILastTransactionsResponse {
+  data: {
+    blockhash: string;
+    blockindex: number;
+    timestamp: number;
+    total: number;
+    txid: string;
+    vin: { addresses: string; amount: number }[];
+    vout: { addresses: string; amount: number }[];
+  }[];
+}
+
+export interface ITransactionProps {
```

Comment:
Actually this interface and `ILastTransactionsResponse` looks the same. Maybe we can change the name of interface from `ILastTransactionsResponse` to `ITransactions` and just re-use it `movement` and `explorer` components? Then we will not need type in line `23`